website/sidebars.js

```diff
@@ -254,6 +254,146 @@ module.exports = {
       label: "Home",
     },
   ],
+  zenith: [
+    {
+      type: "doc",
+      id: "zenith/index",
+      label: "Introduction",
+    },
+    {
+      type: "doc",
+      id: "zenith/quickstart/index",
+      label: "Quickstart",
+    },
+    {
+      type: "category",
+      label: "Using Dagger",
+      collapsible: true,
+      collapsed: true,
+      items: [
+        {
+          type: "doc",
+          label: "Installation",
+          id: "current/cli/install",
+        },
+        {
+          type: "doc",
+          id: "zenith/user/concepts",
+          label: "Concepts",
+        },
+        {
+          type: "doc",
+          id: "zenith/user/best-practices",
+          label: "Best Practices",
+        },
+        {
+          type: "doc",
+          id: "zenith/user/troubleshooting",
+          label: "Troubleshooting",
+        },
+      ],
+    },
+    {
+      type: "category",
+      label: "Programming Dagger",
+      collapsible: true,
+      collapsed: true,
+      items: [
+        {
+          type: "doc",
+          id: "zenith/developer/index",
+        },
+        {
+          type: "category",
+          label: "Quickstarts",
+          collapsible: true,
+          collapsed: true,
+          items: [
+            {
+              type: "doc",
+              label: "Go",
+              id: "zenith/developer/quickstarts/go",
+            },
+            {
+              type: "doc",
+              label: "Python",
+              id: "zenith/developer/quickstarts/python",
+            },
+          ],
+        },
+        {
+          type: "doc",
+          id: "zenith/developer/guides/index",
+        },
+        {
+          type: "doc",
+          id: "zenith/developer/daggerverse",
+        },
+      ],
+    },
+    {
+      type: "category",
+      label: "References",
+      collapsible: true,
+      collapsed: true,
+      items: [
+        {
+          type: "doc",
+          label: "CLI Reference",
+          id: "current/cli/reference",
+        },
+        {
+          type: "link",
+          label: "API Reference 🔗",
+          href: "https://docs.dagger.io/api/reference",
+        },
+        {
+          type: "link",
+          label: "Go SDK Reference 🔗",
+          href: "https://pkg.go.dev/dagger.io/dagger",
+        },
+        {
+          type: "link",
+          label: "Node.js SDK Reference 🔗",
```

Comment:
@slumbering the current Node SDK reference pages are generated with metadata linking to the `current` sidebar. These Zenith pages display a different sidebar `zenith`. So, clicking on the Node SDK reference link from the `zenith` sidebar changes the sidebar to `current`, which creates a confusing experience. We will need to either have a separate generation of the Node SDK reference pages for the Zenith docs with the `zenith` sidebar label, or adjust the Zenith reference page layout to not use a sidebar (similar to what we are doing for the API reference pages).